From 4df84c450d7ce450986e5f1dc5e7ab164b9e0e32 Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@nxp.com>
Date: Fri, 17 Jun 2016 10:04:34 +0530
Subject: [PATCH 59/93] armv8: ls1012a: Make FAT and EXT2 commands support
 common

Makes FAT and EXT2 commands support common for LS1012A QDS,
RDB and FRDM.

Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
---
 include/configs/ls1012a_common.h |    5 +++++
 include/configs/ls1012afrdm.h    |    1 -
 include/configs/ls1012aqds.h     |    6 ------
 include/configs/ls1012ardb.h     |    6 ------
 4 files changed, 5 insertions(+), 13 deletions(-)

diff --git a/include/configs/ls1012a_common.h b/include/configs/ls1012a_common.h
index 2895e7a..7c0e64f 100644
--- a/include/configs/ls1012a_common.h
+++ b/include/configs/ls1012a_common.h
@@ -184,6 +184,11 @@
 #define CONFIG_CMD_BOOTZ
 #define CONFIG_CMDLINE_TAG
 
+/* Used by SCSI, MMC and USB */
+#define CONFIG_CMD_FAT
+#define CONFIG_DOS_PARTITION
+#define CONFIG_CMD_EXT2
+
 #include <asm/fsl_secure_boot.h>
 
 #endif /* __LS1012A_COMMON_H */
diff --git a/include/configs/ls1012afrdm.h b/include/configs/ls1012afrdm.h
index 5e619c1..432317d 100644
--- a/include/configs/ls1012afrdm.h
+++ b/include/configs/ls1012afrdm.h
@@ -38,7 +38,6 @@
 #define CONFIG_SYS_USB_XHCI_MAX_ROOT_PORTS      2
 #define CONFIG_CMD_USB
 #define CONFIG_USB_STORAGE
-#define CONFIG_CMD_EXT2
 
 #define CONFIG_USB_DWC3
 #define CONFIG_USB_DWC3_GADGET
diff --git a/include/configs/ls1012aqds.h b/include/configs/ls1012aqds.h
index cc8e0fc..c06cfbd 100644
--- a/include/configs/ls1012aqds.h
+++ b/include/configs/ls1012aqds.h
@@ -34,8 +34,6 @@
 #define CONFIG_FSL_ESDHC_1_NON_REMOVABLE_CARD
 #define CONFIG_SYS_FSL_MMC_HAS_CAPBLT_VS33
 #define CONFIG_GENERIC_MMC
-#define CONFIG_CMD_FAT
-#define CONFIG_DOS_PARTITION
 #endif
 #endif /* CONFIG_EMU */
 
@@ -45,9 +43,6 @@
 #define CONFIG_SCSI_AHCI
 #define CONFIG_SCSI_AHCI_PLAT
 #define CONFIG_CMD_SCSI
-#define CONFIG_CMD_FAT
-#define CONFIG_CMD_EXT2
-#define CONFIG_DOS_PARTITION
 #define CONFIG_BOARD_LATE_INIT
 
 #define CONFIG_SYS_SATA				AHCI_BASE_ADDR
@@ -204,7 +199,6 @@
 #if defined(CONFIG_HAS_FSL_DR_USB) || defined(CONFIG_HAS_FSL_XHCI_USB)
 #define CONFIG_CMD_USB
 #define CONFIG_USB_STORAGE
-#define CONFIG_CMD_EXT2
 
 #define CONFIG_USB_DWC3
 #define CONFIG_USB_DWC3_GADGET
diff --git a/include/configs/ls1012ardb.h b/include/configs/ls1012ardb.h
index e94df74..f6b3f28 100644
--- a/include/configs/ls1012ardb.h
+++ b/include/configs/ls1012ardb.h
@@ -39,7 +39,6 @@
 #define CONFIG_SYS_USB_XHCI_MAX_ROOT_PORTS      2
 #define CONFIG_CMD_USB
 #define CONFIG_USB_STORAGE
-#define CONFIG_CMD_EXT2
 
 #define CONFIG_USB_DWC3
 #define CONFIG_USB_DWC3_GADGET
@@ -67,8 +66,6 @@
 #define CONFIG_FSL_ESDHC_1_NON_REMOVABLE_CARD
 #define CONFIG_SYS_FSL_MMC_HAS_CAPBLT_VS33
 #define CONFIG_GENERIC_MMC
-#define CONFIG_CMD_FAT
-#define CONFIG_DOS_PARTITION
 #endif
 #endif /* CONFIG_EMU */
 
@@ -78,9 +75,6 @@
 #define CONFIG_SCSI_AHCI
 #define CONFIG_SCSI_AHCI_PLAT
 #define CONFIG_CMD_SCSI
-#define CONFIG_CMD_FAT
-#define CONFIG_CMD_EXT2
-#define CONFIG_DOS_PARTITION
 #define CONFIG_BOARD_LATE_INIT
 
 #define CONFIG_SYS_SATA				AHCI_BASE_ADDR
-- 
1.7.9.5

